config/rbac/manager_role.yaml

```diff
@@ -79,3 +79,15 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
+  - ""
+  resources:
+  - statefulsets
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
```

Comment:
This should have a new line at the end, can you try running `make manifests` and committing the result?